ci/do_ci.sh

```diff
@@ -54,6 +54,9 @@ function bazel_release_binary_build() {
   cp -f "${ENVOY_DELIVERY_DIR}"/envoy "${ENVOY_SRCDIR}"/build_release
   mkdir -p "${ENVOY_SRCDIR}"/build_release_stripped
   strip "${ENVOY_DELIVERY_DIR}"/envoy -o "${ENVOY_SRCDIR}"/build_release_stripped/envoy
+  # TODO(wu-bin): Remove once https://github.com/envoyproxy/envoy/pull/6229 is merged.
+  bazel clean
```

Comment:
Why do we need to `clean`?

Reply:
The "bazel build" in this function happens in the ci workspace, which creates objects in bazel-bin/external/envoy/... ; the 

What appears to be happening is that the "bazel build" in this function and the "bazel build" after this function created same object files in different directories, once in bazel-bin/external/envoy/source/..., and once in bazel-bin/source/...

When the second "bazel build" runs, both instances of object files are given into the linker, causing "multiple definition" errors. [See this link for the actual failure message](https://circleci.com/gh/envoyproxy/envoy/189647?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

The bazel clean hack here removes one copy of the objects, thus avoids the "multiple definition" error.  The hack won't be needed after the ci WORKSPACE is removed.

Reply:
OK, @lizan do you know when we might see #6229 merged? Would be great to avoid having to include this, but if not we can hopefully get a cleanup PR ASAP.

Reply:
I'm completely stuck on that one as the coverage is hard to debug (and not reproducible locally very well), will try something tonight but no ETA for now. The coverage failure seems unrelated to the change though. It would be nice if you can take a look (pushed latest master to the PR).

Reply:
@lizan The [latest coverage run](https://circleci.com/gh/envoyproxy/envoy/193034?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) from your PR failed because some GRPC related test failed. Those tests seems to be flaky, since I've seen the same failures while debugging my PR. I've just started a retest.

@htuch #6229 was stuck on coverage failure, which is likely fixed in this PR, since my change in do_ci.sh's "bazel.coverage" section is very similar to that one.  I think we should get this PR in first.

Reply:
Looks like the coverage test failure in #6229 is not a flake, I have retested it 3 times, the following test fails every time:

SslIpVersionsClientType/GrpcSslClientIntegrationTest.BasicSslRequestWithClientCert/IPv4_EnvoyGrpc